### ISSUE TITLE
Command run from console instead of player

### ISSUE THREAD

**Issue by ShameMate** [Nokorbis/ar-command-signs#51]
I'm trying to use this plugin to run the following command:

/lp user %player% parent set %group%
^ This is a LuckyPerms command.

When a player hits the sign, it runs the command, but seems like the player doesn't have permission. Is there a way to make the sign run the command from "console" instead of the player?

**Comment by 6gav** [comment]
Hey friend, just found this out myself, but if you run it instead as 
>#lp user %player% parent set %group%

with a # instead of a / it will run it in console

**Comment by Nokorbis** [comment]
Yeah, permissions plugins are always a bit tricky because they usually do not let players that are not op to execute the command, so you need to make it run from the console.

**Comment by Nokorbis** [comment]
Please note that every command you add on a sign will require its normal permissions, as usual.
The Sign only makes the Player runs the command.
If the player does not have the permission to run the command it will not work.

This is what Temporary Granted permissions are for.
When the player hits the sign, CommandSigns will grant the player the temporary permissions before running the commands and then remove them [the permissions].
